src/main/java/core/basesyntax/interfaces/FigureInfo.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax.interfaces;
+
+public interface FigureInfo {
+
+    String draw();
```

Comment:
this method should **print** not return information
```suggestion
    void draw();
```